Frends.As2.SendMessage/Frends.As2.SendMessage.Tests/docker/docker-compose.yaml

```
@@ -0,0 +1,31 @@
+    image: greicodex/openas2:latest
+    ports:
+      - 4080:10080
+      - 4081:10081
+      - ${HOST_RESTAPI_PORT:-8443}:8080
+    environment:
+      - OPENAS2PROP_RESTAPI__COMMAND__PROCESSOR__BASEURI=http://0.0.0.0:8080
+      - OPENAS2PROP_RESTAPI__COMMAND__PROCESSOR__USERID=admin
+      - OPENAS2PROP_RESTAPI__COMMAND__PROCESSOR__PASSWORD=admin123
+      - OPENAS2PROP_RESTAPI__COMMAND__PROCESSOR__ENABLED=true
+      - OPENAS2PROP_msg_tracking__enabled=true
+      - OPENAS2PROP_msg_tracking__use_embedded_db=true    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)